services/movie_session.py

```diff
@@ -42,3 +42,8 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
The `get` method can raise a `DoesNotExist` exception if the movie session with the given `id` is not found. Consider using `get_object_or_404` or handling the exception to prevent potential crashes.